common/production_methods/babylon_naval.txt

```diff
@@ -377,29 +393,33 @@ pm_50_pounder_dahlgren_guns = {
 	}
 }
 
-pm_20_pounder_dahlgren_guns = {
+pm_dual_purpose_3_inch_gun = {
```

Comment:
Github f**ed up btw, I can't judge the balance changes because the production method don't align in the diff :/

Reply:
This is probably because I rearranged all the pms, they were an absolute mess before and slowed down every effort I made to change them.